docs/api/rest_api/rest_api_reference/input/examples/shipments/identifier/PATCH/Shipment.json.example

```diff
@@ -0,0 +1,33 @@
+{
+  "Shipment": {
+    "_media-type": "application\/vnd.ibexa.api.Shipment+json",
+    "_href": "\/api\/ibexa\/v2\/shipments\/new_identifier",
+    "id": 1,
+    "identifier": "new_identifier",
+    "status": "prepared",
+    "transitions": {
+      "transitionList": [
+        "send-to-carrier",
+        "cancel"
+      ]
+    },
+    "Order": {
+      "_media-type": "application\/vnd.ibexa.api.Order+json",
+      "_href": "\/api\/ibexa\/v2\/orders\/order\/33f6db52-478f-41f1-afbe-6d38f6a4704f"
+    },
+    "Amount": {
+      "value": "2257",
+      "currency": "EUR"
+    },
+    "createdAt": "2023-06-26T15:40:28+02:00",
+    "updatedAt": "2023-06-26T15:40:28+02:00",
+    "Owner": {
+      "_media-type": "application\/vnd.ibexa.api.User+json",
+      "_href": "\/api\/ibexa\/v2\/user\/users\/14"
+    },
+    "Method": {
+      "_media-type": "application\/vnd.ibexa.api.ShippingMethod+json",
+      "_href": "\/api\/ibexa\/v2\/shipping\/methods\/inpost_parcel_locker_delivery"
```

Comment:
```suggestion
      "_href": "\/api\/ibexa\/v2\/shipping\/methods\/parcel_locker_delivery"
```

Reply:
Done in 63a3945e029aa0ca0c267f22eb073af9140cc761 and/or 5e24f8fddefdd2bd8e0c4e068cb1ca809f9e8fbf